code/modules/power/singularity/singularity.dm

```diff
@@ -359,9 +359,9 @@
 						"<span class='userdanger'>You look directly into the [src.name] and feel weak.</span>")
 	return
 
-
-/obj/anomaly/singularity/proc/emp_area()
-	empulse(src, 8, 10)
+//NSV13
+//obj/anomaly/singularity/proc/emp_area()  //NSV13
+//	empulse(src, 8, 10)  //NSV13
```

Comment:
Commenting this out isn't needed, it's only the actual implementation that's important.

Reply:
So its okay to leave a proc just in the code with no trigger? 
I was under the impression that such was bad, or at least messy.
I modified/commented out the thing that would trigger the emp_area() proc, so I also commented out the proc itself.

Reply:
Normally it's messy, but this isn't new code, and since the functionality has not changed it's worth keeping up to date in case we want to put it back later. It's just not worth adding another merge conflict to the pile.
This also allows admins to still trigger it manually if they want to.